internal/processing/processors/v2alpha1/authorizationpolicy/creator.go

```diff
@@ -26,13 +26,15 @@ type Creator interface {
 	Create(ctx context.Context, client client.Client, api *gatewayv2alpha1.APIRule) (hashbasedstate.Desired, error)
 }
 
-type creator struct{}
+type creator struct {
+	oryPassthrough bool
```

Comment:
I think it would be good to have a comment to explain why this is here and what this should control.
```suggestion
	// Controls that requests to Ory Oathkeeper are also permitted when 
	// migrating from APIRule v1beta1 to v2alpha1.
	oryPassthrough bool
```